pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -238,6 +238,7 @@ func (c *Cluster) makeChownInitContainer(monConfig *monConfig) corev1.Container
 	return controller.ChownCephDataDirsInitContainer(
 		*monConfig.DataPathMap,
 		c.spec.CephVersion.Image,
+		(*corev1.PullPolicy)(&c.spec.CephVersion.ImagePullPolicy),
```

Comment:
All of these should be calling `GetContainerImagePullPolicy()`, right? I see a few of them use the setting directly like this.

Reply:
Yes, will change tht